osgar/drivers/lora.py

```diff
@@ -59,6 +70,7 @@ def __init__(self, config, bus):
         self.raw = b''  # should be defined by Node
         self.last_transmit = None
         self.recent_packets = []
+        self.verbose = False
```

Comment:
This is never set to `True`. What is the expected usage?

Reply:
for replay you can set it via `--verbose` parameter. It should be part of `Node` ... so maybe I should put it there instead?

Reply:
I totally forgot that. No, keep it as is.